### ISSUE TITLE
Unable to install Emulation Station on Ubuntu 15.10

### ISSUE THREAD

**Issue by Yusterlan** [Aloshi/EmulationStation#525]
Hi All,

I've just try to install Emulation Station on Ubuntu 15.10 but it requires libboost-filesystem1.54.0 to do.

I can't install libboost-filesystem1.54.0 since it is not available, but in its way there is a libboost-filesystem1.58.0. I think that the problem is that installer must be modified to recognize this new version of libboost-filesystem

I hope this problem can be solved, I love Emulation Station and I want to use it :)

Regards

**Comment by Yusterlan** [comment]
No way to use this great program in Ubuntu 15.10?

Regards

**Comment by GM-Script-Writer-62850** [comment]
Ubuntu 15.04 has liboost 1.58.0 not 1.54.0
i tried symlinking 1.54.0 to 1.58.0 but i got a error when trying to open es
```
emulationstation: symbol lookup error: emulationstation: undefined symbol: _ZNK5boost6locale9generator8generateERKSs
```

**Comment by GM-Script-Writer-62850** [comment]
I managed to get it working by compiling it
here is the a installer for it
http://evilkitty.duckdns.org/es.ubuntu-15.10-desktop-amd64.tar.bz2
just run the install file with sh and don't forget to enable the 'universe' in software sources first
i did not test it 100%, just made sure it made it's example config file

**Comment by mc707** [comment]
I also tried installing on Ubuntu 15.10 and it gave me the dependency error. When will they fix it?

How different is it installing manually using sh, especially in terms of updating and uninstalling? I have only installed printer drivers using sh, so that wasn't a consideration.

**Comment by GM-Script-Writer-62850** [comment]
the shell script (in that archive i linked) just drops the binary into directly for manually installed packages and puts the data files into the `/etc` directory
the binary i compiled using ubunutu 15.10 so it does not have a issue with the missing dependency as it uses the new version, however that copy will not work (well i doubt it will) on say ubuntu 14.04

if you wanted to remove it just run these 2 commands
```
rm -r /etc/emulationstation
rm /usr/local/bin/emulationstation
```

the install script i made does not install all the dependencies emulationstaton needs, it just installs the ones that don't ship with Ubuntu 15.10

the drawback to a manual install is if you remove a program (with apt) that is the only one that requires say `libfreeimage` the package manager does not know that your manually installed app (emulationstation) even exist and mark `libfreeimage` for auto removal as it is not needed for anything
once `libfreeimage` is removed emualtionstation will not work unless you manually reinstall explicitly

I would give you a deb file but i don't know how to make those

**Comment by mc707** [comment]
I see, thanks for the explanation Script-Writer. I wonder why they don't put it in a Ubuntu repository? Apt-get is so convenient.

**Comment by GM-Script-Writer-62850** [comment]
apt is the preferred way, anyone who knows how to setup a ppa and make a deb package can do that
at least ES is a simple application to install manually

**Comment by gerstrong** [comment]
Yeah, I have to agree. Only compiling it by hand helps. These packages need updates for Ubuntu 15.10

**Comment by 74274265j** [comment]
Can't install directly on Ubuntu 16.04 either.
Tried to install by running 
`sudo dpkg -i emulationstation_amd64_latest.deb`
Tried fixing the dependency issues by using 
`sudo apt-get install -f`
Didn't work.

**Comment by GM-Script-Writer-62850** [comment]
You have 2 choices use a PPA or compile it yourself (flawless compile instructions BTW)
```
sudo add-apt-repository ppa:emulationstation/ppa -y
sudo apt-get update
sudo apt-get install emulationstation-theme-simple emulationstation
```
Note: the ppa puts the theme in /use/share/themes instead of /etc/emulationstation

**Comment by 74274265j** [comment]
It did.
I compiled it and is now installed.
